defaults/rules.polar

```diff
@@ -82,9 +82,9 @@ actor_cannot_access_ca(_: Actor, _: Handle) if false;
 #   2bba. The actor does not have an "inc_cas" attribute (i.e. the actor is not
 #         restricted to certain CAs), _OR_
 #   2bbb. The actor has an "inc_cas" attribute which includes the specified CA
-#         handle (i.e. the CA is included in the set the actor is expicitly
+#         handle (i.e. the CA is included in the set the actor is explicitly
 #         given access to).
-actor_can_access_ca(actor: Actor, ca: Handle) if
+actor_can_access_ca(actor: Actor, ca: PolarHandle) if
```

Comment:
Changing this class name is potentially breaking for anyone using their own policy files. Why not either still call it `Handle` (even though it is now a New Type because the underlying `Handle` type has moved out of the crate) or use [`ClassBuilder::name()` ](https://docs.rs/oso/0.12.4/oso/struct.ClassBuilder.html#method.name) to call it `Handle` as far as Oso is concerned, and thus avoid breakage in policy files and to actually refer to it by its real name instead of a "wrapper" name that only exists due to Rusts limitations about implementing traits on external types?

Reply:
Also, the references to `Handle` in other `.polar` files have not been updated, e.g. see `abac.polar`.

Reply:
Done! Good point.